src/libPMacc/include/particles/ParticleDescription.hpp

```diff
@@ -42,13 +42,20 @@ namespace PMacc
  * @tparam T_MethodsList sequence or single class with particle methods
  *                       (e.g. calculate mass, gamma, ...)
  *                       (e.g. useSolverXY, calcRadiation, ...)
+ * @tparam T_FrameExtensionList sequence or single class with frame extentions
+ *                    - extension must be a unary template class that supports bmpl::apply1<>
+ *                    - type of the finale frame is applied to every extension class
+ *                      ( this allows pointer and references in a frame to it self)
+ *                    - the finale frame which use ParticleDescription inherits from all
```

Comment:
the finale frame that uses ParticleDescription inherits from all
extension classes